tests/test_cli.py

```diff
@@ -181,6 +181,8 @@ def test_node_runs_gateway(opts, expected):
     "args",
     [
         ([MOCK_HASH, "--network", "goerli"]),
+        ([MOCK_HASH, "--network", "goerli2"]),
+        ([MOCK_HASH, "--network", "integration"]),
```

Comment:
Since these aren't yet supported by the `STARKNET_ENVIRONMENT`, the subprocess call includes the explicit `--feeder_gateway_url`. These tests should pass if you add the url to `expected` for the goerli2 and integration network tests.